pygsti/algorithms/fiducialpairreduction.py

```diff
@@ -305,7 +300,7 @@ def _get_number_amplified(m0, m1, len0, len1, verb):
     return listOfAllPairs
 
 def find_sufficient_fiducial_pairs_per_germ(target_model, prep_fiducials, meas_fiducials,
-                                            germs, pre_povm_tuples="first",
+                                            germs, prep_povm_tuples="first",
```

Comment:
Excellent point, thanks for catching that! We should mentally bookmark this whenever we get around to exposure marking this file at the function-level. My thinking here was mostly that the naming of this kwarg was a minor pet peeve since most (including me when I go a long time without looking at this code since we rarely if ever change this in practice) would read this as 'pre-POVM' (which is an incorrect and confusing descriptor) rather than what is really intended.